product_docs/docs/pem/9/monitoring_performance/notifications.mdx

```diff
@@ -130,11 +134,37 @@ Use the **Payload** tab to define the JSON data to send to the endpoint when an
 
 -   Select **Test Connection** to test notification delivery to the mentioned endpoint.
 
+### Defining webhook alert levels
+
 Use the **Notifications** tab to specify an alert level for webhook endpoints:
 
 -   Set **All alerts** to **Yes** to enable all alert levels to send notifications.
 -   To send a notification when a specific alert level is reached, set the slider next to an alert level to **Yes**. You must set **All alerts** to **No** to configure an individual alert level.
 
+### Example: sending notifications to Slack
+
+In Slack, follow the instructions in [Getting started with incoming webhooks](https://api.slack.com/messaging/webhooks) to:
+- Create a Slack app
```

Comment:
```suggestion
- Download an login to a Slack app
```

Reply:
Creating a slack app is correct. Slack apps live in Slack server side and are created on the web UI, there is nothing to download.

Reply:
Existing wording is correct as pointed out by DJ.